webui/react/src/types.ts

```diff
@@ -154,6 +154,7 @@ export const DetailedUser = t.intersection([
   User,
   t.partial({
     agentUserGroup: AgentUserGroup,
+    isPasswordWeak: t.boolean,
```

Comment:
Not necessary for the warning toast itself, but might be used in the near future as described [here](https://hpe-aiatscale.atlassian.net/browse/DFR-483?focusedCommentId=30954)